dev-tools/.beatconfig

```diff
@@ -1,4 +1,4 @@
 packetbeat-/packetbeat-
```

Comment:
Does anyone even know what this file is for? I think it can be removed.

Reply:
It's coming from https://github.com/elastic/beats/pull/1439 and was linked to the old way of important dashboard so it can be safely removed.

Reply:
My preference is that removals like that be done outside of a spelling PR. Before or after doesn't matter to me, but it's really problematic for people looking at a huge commit (and my understanding is that elastic tends to squash) to discover that in addition to doing a rather "simple" and "harmless" thing, it also did something else.

One positive side effect of spell checking is finding bugs and stale content.

That said, it's a personal preference. Within limits, I'll do as requested...